File: 프로그래머스/1/12901. 2016년/2016년.java
class Solution {
    public String solution(int a, int b) {
        int[] arr = {0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        String[] day = {"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};
        
        int dayNum = 0;
        for (int i = 1; i < a; i++) {
            dayNum += arr[i];
        }
        
        dayNum += b;
        
        String answer = day[dayNum % 7];
        
        return answer;
    }
}

File: 프로그래머스/1/12901. 2016년/2016년.cs
public class Solution {
    public string solution(int a, int b) {
        int[] days = {0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        string[] s = {"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};
        
        int dayNum = 0;
        
        for (int i = 1; i < a; i++) {
            dayNum += days[i];
        }
        
        dayNum += b;
        
        return s[dayNum % 7];
    }
}